_utilities/layout-grid.md

```diff
@@ -504,116 +357,173 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
-  'card':              false,   // 160px
-  'card-lg':           false,   // 240px
-  'mobile':            false,   // 320px
-  'mobile-lg':         true,    // 480px
-  'tablet':            true,    // 640px
-  'tablet-lg':         false,   // 880px
-  'desktop':           true,    // 1024px
-  'desktop-lg':        false,   // 1200px
-  'widescreen':        false,   // 1400px
-);
-{% endhighlight %}
-
-### Mixins
-Mixins can be used in conjunction with grid variables to add grid functionality to semantic component Sass.
-
-{% highlight scss %}
-// Creates a wrapper for a series of rows
-// $container-size can be mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, or widescreen
-@include grid-container;
-@include grid-container($container-size);
-
-// Creates a wrapper for a series of columns
-@include grid-row;
+## Theme settings
+You can redefine column gap token sizes, set the grid container max width, and update site margin by defining the relevant theme setting in your `_uswds-theme.scss` file.
 
-// Specify the width between columns
-// $gap-size can be sm, md, lg, 2px, 05, 1, 3, 4, 6
-@include grid-gap;
-@include grid-gap($gap-size);
+Visit the [settings page]({{ site.baseurl }}/documentation/settings) for more information and available settings.
 
-// Make the element full-width
-@include u-width(full);
+<section id="utility-mixins" class="padding-top-4">
+  <h2 class="site-h2 margin-y-0">Utility mixins</h2>
+  {% include utilities/utility-mixin-intro.html %}
 
-// Specify the number of columns the element should span
-// $columns can be auto, fill, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12
-@include grid-col;
-@include grid-col($columns);
-
-// Get fancy by offsetting or changing the display order
-// $offset can be none, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11
-@include grid-offset($offset);
-
-// $order can be first, last, initial, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11
-@include u-order($order);
-{% endhighlight %}
-
-### Example usage
-You can modify the variables to your custom values or just use the mixins with their default values. Here’s an example of using the default settings to create a two-column layout with a gap.
-
-{% highlight scss %}
-.example-container {
-  @include grid-container;
-}
-
-.example-row {
-  @include grid-row;
-
-  // Add column gaps
-  &.content-row {
-    @include grid-gap;
-  }
-}
-
-.example-content-main {
-  @include u-width(full);
-
-  @include at-media(tablet) {
-    @include grid-col(6);
-  }
-
-  @include at-media(desktop) {
-    @include grid-col(8);
-  }
-}
+  <table class="usa-table--borderless site-table-responsive site-table-simple">
+    <thead>
+      <tr>
+        <th scope="col" class="tablet:maxw-card-lg display-inline-flex flex-2">Utility</th>
+        <th scope="col" class="display-inline-flex flex-2">Mixin</th>
+        <th scope="col" class="display-inline-flex flex-1">Accepted values</th>
+        <th scope="col" class="display-inline-flex flex-2">Example</th>
```

Comment:
@amyleadem This is a really good point. I originally wrapped the parenthesis in the code tags to _hopefully_ demonstrate that it was optional but I don't think that did the trick.

After looking at the other utility pages again, it looks like most rely on a tokens table with visual examples rather than an `accepted values` column in their mixin tables. 

For reference, here's how [flex guidance](https://designsystem.digital.gov/utilities/flex/) handles flex wrap which doesn't have a design token page to show available units:

![image](https://github.com/user-attachments/assets/9f08586f-9ea4-4330-924e-dc8579e1c936)

![image](https://github.com/user-attachments/assets/c5477dd8-87e3-438c-a310-9c1c72465cb2)

I originally opted to maintain the guidance at the top of the page because of the increased scope. Bringing it up in case it's something we'd like to explore down the road.

For now, I opted to follow your recommendation but named the header `mixin` to mirror the `utility` column.

Resolved in [7af5f03](https://github.com/uswds/uswds-site/pull/2276/commits/7af5f034089632d76c15e6525bc87b8b9fa49ae0)